src/client/components/app/Transactions/WithdrawCreditTx.tsx

```diff
@@ -49,9 +49,17 @@ export const WithdrawCreditTx: FC<BorrowCreditProps> = (props) => {
       setErrors([...errors, 'no selected position']);
       return '0';
     }
-    let maxWithdraw: string = `${Number(selectedPosition.deposit) - Number(selectedPosition.principal)}`;
-    maxWithdraw = normalize('amount', maxWithdraw, selectedPosition.token.decimals);
-    return maxWithdraw;
+    const maxWithdrawAmount: number =
+      Number(selectedPosition.deposit) - Number(selectedPosition.principal) + Number(selectedPosition.interestRepaid);
```

Comment:
feel like it should be BigNumber not Number right?

Reply:
TBD

Reply:
fixed